.pre-commit-config.yaml

```diff
@@ -20,7 +20,7 @@ repos:
     rev: 19.10b0
     hooks:
     - id: black
-      language_version: python3.6
+      language_version: python3.7
```

Comment:
this is what changed the file below, because dataclasses were not part of 3.6 > added as a dependency -> means that import order needs to be changed (pure python and deps are not sorted the same way) -> broken lint when isort against 3.7